app/Models/Dinnerform.php

```diff
@@ -61,6 +62,12 @@ public function event()
         return $this->belongsTo('Proto\Models\Event');
     }
 
+    /** @return BelongsTo */
+    public function orderedBy()
+    {
+        return $this->belongsTo('Proto\Models\User', 'ordered_by');
+    }
```

Comment:
both this method and the database column result in the ordered_by property. It would be best to have different names for the database column and this method. For example a user_id column or ordered_by_user_id column name would work.